extensions/email/src/main/java/org/mule/extension/email/api/predicate/DefaultEmailPredicateBuilder.java

```diff
@@ -30,12 +31,12 @@
  * None of the criteria fields are mandatory. If a particular criteria is not specified, then it's simply not applied on the
  * evaluation.
  * <p>
- * The class is also given the &quot;matcher&quot; alias to make it DSL/XML friendly.
+ * The class is also given the &quot;default-matcher&quot; alias to make it DSL/XML friendly.
  *
  * @since 4.0
  */
-@Alias("matcher")
-public class EmailPredicateBuilder {
+@Alias("default-matcher")
+public class DefaultEmailPredicateBuilder {
```

Comment:
I don't understand this... What's a default matcher and where would I use it? Sounds like this class should have no alias, be made abstract and renamed to simply EmailPredicateBuilder